opentreemap/treemap/js/src/buttonEnabler.js

```diff
@@ -52,9 +55,15 @@ function fullyEnable($el, href) {
 }
 
 function fullyDisable($el, disabledTitle) {
+    var wrapperTemplate =
+            '<label ' + disabledButtonWrapperAttrPair +
+            ' data-placement="bottom" title="%s"></label>';
```

Comment:
I don't think we need `data-placement="bottom"`.  Is that a vestigial Bootstrap thing?